### ISSUE TITLE
Find me email button does not work

### ISSUE THREAD

**Issue by NealMatta** [NealMatta/personal-website#1]
![image](https://github.com/NealMatta/personal-website/assets/12685798/84444e92-c0b4-42a3-b4e2-8dea8556c4dc)
- A new tab opens up, but nothing happens
- a potential solution can be to change that icon to copy the email

**Comment by AK97** [comment]
This is almost certainly a personal browser settings problem. [The link your button opens is correct](https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_link_mailto). Works on my system/browser.

Look up how to change ```mailto``` settings for your browser to fix your issue.

**Comment by NealMatta** [comment]
Good catch. Regardless, I'd like to change it to an email copied feature. I haven't worked a lot with css animations and this seems like a good change to get my feet wet
